syntax = "proto3";

option go_package = "/DH_params";

message DH_params_Request {
  string nonce        = 1;
  string server_nonce = 2;
  int32  message_id   = 3;
  int32  a            = 4;
}

message DH_params_Response {
  string nonce        = 1;
  string server_nonce = 2;
  int32  message_id   = 3;
  int32  b            = 4;

}

service DH_params_Service {
  rpc ProcessRequest (DH_params_Request) returns (DH_params_Response);
}
